cluster-scope/overlays/ocp-staging/kustomization.yaml

```diff
@@ -22,6 +24,7 @@ resources:
 patches:
   - path: apiservers/cluster_patch.yaml
   - path: oauths/cluster_patch.yaml
+  - path: groups/koku-metrics-operator-admin.yaml
```

Comment:
If this is a patch, then you need to include the group in the `resources` section.

Reply:
not sure which group you are talking about. I included `../../base/user.openshift.io/groups/koku-metrics-operator-admin` in resources in this PR.

Reply:
But it's not listed in the `cluster-scope/overlays/ocp-staging/kustomization.yaml`, so there's nothing to patch.

Reply:
I think I am simply blind.